conference.js

```diff
@@ -553,6 +486,105 @@ export default {
      * Whether the local participant is the dominant speaker in the conference.
      */
     isDominantSpeaker: false,
+
+    /**
+     * Creates local media tracks and connects to a room. Will show error
+     * dialogs in case accessing the local microphone and/or camera failed. Will
+     * show guidance overlay for users on how to give access to camera and/or
+     * microphone,
+     * @param {string} roomName
+     * @param {boolean} startScreenSharing - if <tt>true</tt> should start with
+     * screensharing instead of camera video.
+     * @returns {Promise.<JitsiLocalTrack[], JitsiConnection>}
+     */
+    createInitialLocalTracksAndConnect(roomName, startScreenSharing) {
```

Comment:
Looks like this one fell through the cracks:

> Thinking ahead, this should be an options object, since we already have plans to have more options, such as startAudioOnly, right?